src/modelling_utilities/split_dynamic_images.cxx

```diff
@@ -0,0 +1,98 @@
+//
+//
+/*
+  Copyright (C) 2018, University College London
+  This file is part of STIR.
+
+  This file is free software; you can redistribute it and/or modify
+  it under the terms of the GNU Lesser General Public License as published by
+  the Free Software Foundation; either version 2.1 of the License, or
+  (at your option) any later version.
+
+  This file is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU Lesser General Public License for more details.
+
+  See STIR/LICENSE.txt for details
+*/
+
+/*!
+  \file
+  \ingroup utilities
+  \brief Split dynamic image into individual images
+  \author Richard Brown
+
+  \par Usage:
+  \code 
+  split_dynamic_images output_prefix input output_format
```

Comment:
`output_format`is maybe a bit cryptic. `output_parameter_file`? Ideally also include a sample of a file in the doxygen. Could make it optional (but feel free to just put a \todo)